lib/feature_requests.rb

```diff
@@ -0,0 +1,38 @@
+module Lita
+# A class for dealing with the feature request board on trello
+  class FeatureRequests
+
+    SEVEN_DAYS = (60*60*24*7)
+
+    def initialize(trello_client)
+      @trello_client = trello_client
+    end
+
+    # Select cards created within the previous seven days
+    #
+    def to_msg(board_id)
+      board = @trello_client.find(:boards, board_id)
+      board.cards.select{ |card|
+        card.actions.last.date > seven_days_in_seconds
+      }.map{ |card|
+        "#{card.name}, #{card.short_url}, #{card.list.name}"
+      }.join("\n")
+    end
+
+    # Returns a list of all cards on the Feature request board
+    #
+    def all_feature_request(board_id)
```

Comment:
Oh my gosh how embarrassing. Now I've _really_ fixed it, amended the commit and pushed it in.